src/ol_infrastructure/applications/edxapp/__main__.py

```diff
@@ -1018,7 +1018,11 @@
 
 edxapp_acm_cert_validation_records = (
     edxapp_web_acm_cert.domain_validation_options.apply(
-        partial(acm_certificate_validation_records, zone_id=edxapp_zone_id)
+        partial(
```

Comment:
I know this is just a formatting change, but do we want this to be in the PR?

Reply:
We do, because I actually changed the signature of the method being used, which is what caused the formatting to get bumped to multiple lines.